app/invocation/workflow_rerun_button.tsx

```diff
@@ -65,7 +65,6 @@ export default class WorkflowRerunButton extends React.Component<WorkflowRerunBu
 
     const configuredEvent = this.props.model.workflowConfigured;
 
-    // TODO(Maggie): Deprecate ExecuteWorkflowResponse.InvocationID field
```

Comment:
is this meant for a different PR?

Reply:
No I just forgot to delete this comment earlier. The code has already been cleaned up